site/content/in-dev/unreleased/overview.md

```diff
@@ -180,6 +180,12 @@ To secure interactions with service connections, Polaris vends temporary storage
 execution. These credentials allow the query engine to run the query without requiring access to your external cloud storage for
 Iceberg tables. This process is called credential vending.
 
+As of now, the following limitations are known regarding Apache Iceberg support:
+
+- **remove_orphan_files:** This operation does not currently support 
+  credential vending when utilizing Apache Spark as the compute engine.
+  - For more details, please refer to [apache/iceberg#7914](https://github.com/apache/iceberg/pull/7914).
```

Comment:
Should we be using the new Hugo-style links? I am honestly not clear on that

Reply:
@eric-maynard is right, I think it's better to use Hugo style link to have a clean documentation.

Reply:
Thanks for the info. Will finish validations later today then update the PR with suggestions above.

Reply:
From some quick read, Hugo style link is mainly for internal links within the site and external links seems to be preferred as markdown's style instead. Also, there are many places we are using markdown style links for external reference across current site. If that is more preferred, should we do it in a different PR instead?
 
Then in terms of table maintenance operations validation with credential vending, following are tested:

Trino:
 - optimize: passed
 - expire_snapshots: passed
 - remove_orphan_files: passed
 - drop_extended_stats: passed

Spark:
- expire_snapshots: passed
- remove_orphan_files: failed (not supported as mentioned in this PR)
- rewrite_data_files: passed
- rewrite_manifests: passed
- rewrite_position_delete_files: passed

Reply:
@MonkeyCanCode thanks a lot for validating maintenance procedures. It's a good callout, but it is essentially an Spark issue. Maybe we can reword to something like this:
```
- **remove_orphan_files:** Apache Spark can't use credential vending for this due to a known issue. See [apache/iceberg#7914](https://github.com/apache/iceberg/pull/7914) for details. 
```

Reply:
Anytime. Updated.